app/models/user.js

```diff
@@ -0,0 +1,24 @@
+module.exports = (sequelize, DataTypes) => {
+  const User = sequelize.define('users', {
+    firstName: {
```

Comment:
I would specify the id

`id: { type: DataTypes. INTEGER, primaryKey: true, autoIncrement: true },`